### ISSUE TITLE
Add subdural hygroma

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10555]
**Preferred term label:**
Subdural hygroma

**Synonyms**
Subdural hygromata
**Definition (free text, please give PubMed ID)**
The accumulation of cerebral spinal fluid in the subdural space PMID: 9653522

**Parent term (use hpo.jax.org/app)**
Abnormal brain morphology HP:0012443

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Trauma PMID: 9653522
Cranioectodermal dysplasia (omim614378) PMID28621010

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl
Adding new term: 
- ID: HP:6001070
- label: Subdural hygroma
- parents: [Abnormal meningeal morphology (HP:0010651)](https://hpo.jax.org/app/browse/term/HP:0010651)
- pmid: PMID:9653522
- def: Accumulation of cerebral spinal fluid in the subdural space.
- comment: 
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10555
- synonyms:
